dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -3261,7 +3273,51 @@ async fn cmd_db_instance_info(
         println!("{table}");
     }
 
-    if history {
+    if resources || all {
+        use db::schema::virtual_provisioning_resource::dsl as resource_dsl;
+        let mut resource = resource_dsl::virtual_provisioning_resource
+            .filter(resource_dsl::id.eq(id.into_untyped_uuid()))
+            .limit(1)
```

Comment:
I would remove the limit and instead print a warning if there are multiple entries for a single instance--shouldn't ever happen, but we don't want the debugger to hide it from us if it does. (If we do have more than one, printing them all along with their modified times seems like it'd be valuable, since that gives the reader a timestamp to use to find the relevant points in the Nexus logs.)

Reply:
Hmm, although now that I think about it, we should be covered here by the use of the instance ID as the resource table key--it's the *collection* table where we've had trouble in the past, and there it was because we were doing some arithmetic badly. If that's so, the `limit` bit in this query still seems superfluous, but no need to try to rework any of the other display logic to handle the more-than-one-entry case.

Reply:
The UUID is a primary key, right? https://github.com/oxidecomputer/omicron/blob/ccaa9ca4657765260974878f5d8e0b973bc6a0f7/schema/crdb/dbinit.sql#L499-L501

So, shouldn't the database guarantee that there would only ever be one entry returned regardless?

Reply:
In any case, I'm happy to remove the `limit` clause, I just added it because I thought I saw other OMDB commands that query a table by its primary key adding a similar `limit(1)` and wanted to be consistent.